client/my-sites/site-settings/custom-content-types/index.jsx

```diff
@@ -52,6 +53,13 @@ class CustomContentTypes extends Component {
 		return isRequestingSettings || isSavingSettings;
 	}
 
+	requestSitePostTypes = () => {
+		const { siteId } = this.props;
+		if ( siteId ) {
+			this.props.requestPostTypes( siteId );
```

Comment:
I'm having trouble finding whereabouts `requestPostTypes` is bound to `dispatch`?